### ISSUE TITLE
Truncated remaining battery indication

### ISSUE THREAD

**Issue by e7d** [NicoHood/arc-theme#43]
### Details

- Distribution: Ubuntu 17.10, Kernel 4.13.0-25-generic
- Desktop environment: Wayland 2:1.19.5-0ubuntu2
- Arc version/commit: master@f6284cc0caaa5859659baf5db38f56d60cabca28 
- GTK+ 3 version: 3.22.25-0ubuntu0.1
  - Murrine engine version: 0.98.2-0ubuntu5
  - `gnome-themes-standard` version: 3.22.3-1ubuntu1
- Installation method: source

### Description

Using a fresh installation of Ubuntu, roughly 1 week, I'm getting an issue with the "system menu" (don't know the official name) that you get when clicking the top right icons. As you can see the battery remaining time indication is truncated, making it impossible to know the remaining percentage of the battery :
![Truncated remaining battery indication](https://user-images.githubusercontent.com/1412033/35046856-8adcad9a-fb98-11e7-80d4-34e38c34428f.png)

Just in case, here are my settings regarding appearance, as seen on Tweaks app :
![Tweaks appearance settings](https://user-images.githubusercontent.com/1412033/35046903-ac8decec-fb98-11e7-8b39-97fd49ddad93.png)

Feel free to ask me for any additional information or tell me if I'm missing anything obvious.

**Comment by fossfreedom** [comment]
that looks like an ellipsis - usually that happens when the field is a fixed size and the font size causes the additional characters that normally would expand in that field instead be displayed as an ellipsis.  

To test the theory make the font sizes slightly smaller

**Comment by e7d** [comment]
Agree about the ellipsis, especially since GTX themes looks like built around HTML-CSS based code.

As suggested, I tried to size down all the font sizes from 11-11-11-13 to 6-6-6-8, but with no success, be it after a Gnome Shell restart with "killall -HUP gnome-shell" or a full reboot :
![small-fonts](https://user-images.githubusercontent.com/1412033/35092671-6bbe2576-fc40-11e7-8026-8607e90dce4e.png)

I'm looking after another solution though, which would be better I guess.
The current display seems to follow this format: `H:mm Remaining (XX%)`.
It would be great if it could be changed to: `XX% (H:mm left)` 

In my opinion, it would be more effective as both critical information, percentage and time left, would be positioned earlier in the chain, so they would always be visible, be it truncated by an ellipsis or not.
It could look like this:
![battery-left-preview](https://user-images.githubusercontent.com/1412033/35094254-06361524-fc45-11e7-8da3-f6d82741f675.png)

**Comment by NicoHood** [comment]
Does the issue still exist? Here on ArchLinux with gnome 3.28 (Xorg) I have enough space in the menu. I cannot do a screenshot as its blocked when opening this menu.

**Comment by e7d** [comment]
Yes it is. You can find below a screenshot I did with my current desktop configuration, after having updated the different dependencies as long as your theme.
![image](https://user-images.githubusercontent.com/1412033/39296819-3020206a-4942-11e8-99b4-b4ac52bfd437.png)

I'm using a Dell XPS 15 with 4K screen, but currently configured as 1080p because of scaling issues when I use my external monitor (but that's another story):
![image](https://user-images.githubusercontent.com/1412033/39296982-a1541778-4942-11e8-8ba5-fe25288977cb.png)

Let me know if I can be of any more use. I'll be glad to help.

**Comment by NicoHood** [comment]
Do other themes also have this bug?

**Comment by shoeper** [comment]
I do have the same issue. It has not been there with the default theme on Fedora.
